src/concept/answer/ConceptMap.jl

```diff
@@ -4,12 +4,42 @@ struct ConceptMap
     data::Dict{String, AbstractConcept}
 end
 
-# Java uses "unmodifiableMap" but if that's the case I would probably hide
-# the implementation here and not expose the underlying Dict. Hence commenting
-# out the following function.
-# map(cm::ConceptMap) = cm.data
+struct Explainable
+    conjunction::String
+    explainable_id::Int64
+end
+
+mutable struct Explainables
+    relations::Optional{Dict{String,Explainable}}
+    attributes::Optional{Dict{String,Explainable}}
+    ownerships::Optional{Dict{Tuple{String},Explainable}}
+end
+
+function Explainable(explainable::Proto.Explainable)
+   return Explainable(explainable.conjunction, explainable.id)
+end
+
+Explainable() = Explainables(nothing,nothing,nothing)
```

Comment:
```suggestion
Explainable() = Explainable(nothing, nothing, nothing)
```

Reply:
Done

Reply:
Here Explainables was meant.  The right statements should be:
Explainables() = Explainables(nothing, nothing, nothing)

Reply:
That's true haha

Reply:
It just looks a bit strange, since usually you'll have a module or package with the plural name, and a type/struct name that is singular.

Reply:
Thats an implementation detail from the python and java client where I didn't want change the naming. This would be a discussion point with the vatical guys. I changed on some places the blue print from java or python. But this is more for internal mechanisms than for api things.